File: firefly-example/src/main/java/com/firefly/example/reactive/coffee/store/vo/UserInfo.java
package com.firefly.example.reactive.coffee.store.vo;

import com.firefly.$;

/**
 * @author Pengtao Qiu
 */
public class UserInfo {
    private Long id;
    private String name;

    public Long getId() {
        return id;
    }

    public void setId(Long id) {
        this.id = id;
    }

    public String getName() {
        return name;
    }

    public void setName(String name) {
        this.name = name;
    }

    @Override
    public String toString() {
        return $.json.toJson(this);
    }
}

File: firefly-example/src/main/kotlin/com/firefly/example/kotlin/coffee/store/vo/UserInfo.kt
package com.firefly.example.kotlin.coffee.store.vo

import com.firefly.kotlin.ext.annotation.NoArg
import java.io.Serializable

/**
 * @author Pengtao Qiu
 */
@NoArg
data class UserInfo(
    var id: Long,
    var name: String
                   ) : Serializable